src/main/java/teammates/logic/core/NotificationsLogic.java

```diff
@@ -36,6 +39,15 @@ public NotificationAttributes getNotification(String notificationId) {
         return notificationsDb.getNotification(notificationId);
     }
 
+    /**
+     * Gets a list of notifications.
+     *
+     * @return a list of notifications with the specified {@code notificationIds}.
```

Comment:
Should this be targetUser? Seems like you are querying by that, unless this comment is trying to say something else.